newIDE/app/src/MainFrame/EditorTabs/UseEditorTabsStateSaving.js

```diff
@@ -155,8 +155,10 @@ const useEditorTabsStateSaving = ({
         );
       }
 
+      let openedEditorsCount = 0;
       for (const editorOpeningOption of editorsOpeningOptions) {
         newEditorTabs = openEditorTab(newEditorTabs, editorOpeningOption);
+        openedEditorsCount++;
```

Comment:
Is `openedEditorsCount` the same as `editorsOpeningOptions.length`?

Reply:
Indeed, will change it